ProjFS.Mac/PrjFSKext/public/Message.h

```diff
@@ -47,25 +55,19 @@ struct MessageHeader
     int32_t             pid;
     char                procname[MAXCOMLEN + 1];
 
-    // Size of the flexible-length, nul-terminated path following the message body, including the nul character.
-    uint16_t            pathSizeBytes;
+    // Sizes of the flexible-length, nul-terminated paths following the message body.
+    // Sizes include the nul characters but can be 0 to indicate total absence.
+    uint16_t            pathSizesBytes[MessagePath_Count];
```

Comment:
This is now an array instead of 2 scalars or a union of an array and a struct with 2 scalars.

Reply:
I personally like the explicit from/to, but I know you prefer it the other way.  Can we get a third opinion from @wilbaker?  Both ways certainly work :-), I prefer explicit fields for 2 variables rather than an array.  I think it's safer.